include/seqan/modifier/modifier_position.h

```diff
@@ -0,0 +1,513 @@
+
+template <typename THost, typename TPositions>
+struct Difference<ModifiedString<THost, ModPos<TPositions> > > : Difference<TPositions> {};
+
+// ----------------------------------------------------------------------------
+// Metafunction Size
+// ----------------------------------------------------------------------------
+
+template <typename THost, typename TPositions>
+struct Size<ModifiedString<THost, ModPos<TPositions> > > : Size<TPositions> {};
+
+// ----------------------------------------------------------------------------
+// Metafunction Position
+// ----------------------------------------------------------------------------
+
+template <typename THost, typename TPositions>
+struct Position<ModifiedString<THost, ModPos<TPositions> > > : Position<TPositions> {};
+
+// ----------------------------------------------------------------------------
+// Metafunction Iterator
+// ----------------------------------------------------------------------------
+
+template <typename THost, typename TPositions>
+struct Iterator<ModifiedString<THost, ModPos<TPositions> >, Standard>
+{
+    typedef Iter<ModifiedString<THost, ModPos<TPositions> >, PositionIterator>   Type;
+};
+
+template <typename THost, typename TPositions>
+struct Iterator<ModifiedString<THost, ModPos<TPositions> > const, Standard>
+{
+    typedef Iter<ModifiedString<THost, ModPos<TPositions> > const, PositionIterator>   Type;
+};
+
+template <typename THost, typename TPositions>
+struct Iterator<ModifiedString<THost, ModPos<TPositions> >, Rooted>
+{
+    typedef Iter<ModifiedString<THost, ModPos<TPositions> >, PositionIterator>   Type;
+};
+
+template <typename THost, typename TPositions>
+struct Iterator<ModifiedString<THost, ModPos<TPositions> > const, Rooted>
+{
+    typedef Iter<ModifiedString<THost, ModPos<TPositions> > const, PositionIterator>   Type;
+};
+
+// ----------------------------------------------------------------------------
+// Metafunction Prefix
+// ----------------------------------------------------------------------------
+
+//template <typename THost, typename TPositions>
+//struct Prefix<ModifiedString<THost, ModPos<TPositions> > >
+//{
+//    typedef ModifiedString<THost, ModPos<TPositions> >    Type;
+//};
+//
+//template <typename THost, typename TPositions>
+//struct Prefix<ModifiedString<THost, ModPos<TPositions> > const> :
+//    Prefix<ModifiedString<THost, ModPos<TPositions> > > {};
+
+// ----------------------------------------------------------------------------
+// Metafunction Suffix
+// ----------------------------------------------------------------------------
+
+//template <typename THost, typename TPositions>
+//struct Suffix<ModifiedString<THost, ModPos<TPositions> > >
+//{
+//    typedef ModifiedString<THost, ModPos<TPositions> >    Type;
+//};
+//
+//template <typename THost, typename TPositions>
+//struct Suffix<ModifiedString<THost, ModPos<TPositions> > const> :
+//    Suffix<ModifiedString<THost, ModPos<TPositions> > > {};
```

Comment:
Commented out intensionally, or can it be removed? If not please document why it is commented out.